pkg/addons/data/Deployment.apps/openshift-infra/customer-admin-reconciler.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: customer-admin-reconciler
+  namespace: openshift-infra
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: customer-admin-reconciler
+  template:
+    metadata:
+      labels:
+        app: customer-admin-reconciler
+    spec:
+      serviceAccountName: customer-admin-reconciler
+      containers:
+      - args:
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          cat <<'SCRIPT' >/tmp/reconcile
+          #!/bin/bash -x
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          project=$1
+          echo "Project: ${project}"
+
+          # do nothing for whitelisted projects
+          for ns in '^openshift-.*$' '^openshift$' '^kube-.*$' '^default$';
```

Comment:
This is incomplete, enforce:

```go
ForbiddenNames    = []string{"openshift", "kubernetes", "kube", "default"}
ForbiddenPrefixes = []string{"openshift-", "kubernetes-", "kube-"}
```

Reply:
Ignore any namespace that equals `ForbiddenNames` or starts with `ForbiddenPrefixes`.

Reply:
Added 'kubernetes' and 'kubernetes-'.